game/scripts/npc/abilities/dark_seer_ion_shell.txt

```diff
@@ -59,7 +59,7 @@
       "03"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "duration"                                        "25"
+        "duration"                                        "21 24 27 30 33 36"
```

Comment:
"DOTA_UNIT_TARGET_FLAG_NOT_CREEP_HERO" is it cool to remove this? ion shell can be cast on creep heroes now.

Reply:
I removed it.